tests/assets/ansible/roles/node-base/tasks/sles-15.yml

```diff
@@ -0,0 +1,42 @@
+---
+# FIXME(toabctl): Allow usage of SCC and official channels for repos
+- name: add basic sle15 repositories
+  block:
+    - name: add basic repositories
+      zypper_repository:
+        name: "{{ repo.name }}"
+        repo: "{{ repo.url }}"
+        state: present
+        auto_import_keys: yes
+        overwrite_multiple: yes
+        runrefresh: yes
+      loop:
+        - { name: "basesystem_pool", url: "http://download.suse.de/ibs/SUSE/Products/SLE-Module-Basesystem/{{ sle_major }}-SP{{ sle_minor }}/x86_64/product/" }
```

Comment:
(nit) Can we indent this nicer? like have name/url on separate lines?

Reply:
Done